pyproject.toml

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Contributors to the OpenEXR Project.
 
 [build-system]
-requires = ["scikit-build-core==0.10.7", "pybind11"]
+requires = ["scikit-build-core==0.10.7", "pybind11", "numpy"]
```

Comment:
Numpy is not needed at build time. Pybind11 doesn't need numpy and will not use numpy.